.bumpversion.cfg

```diff
@@ -1,9 +0,0 @@
-[bumpversion]
```

Comment:
@adamhadani thanks for your reply! actually, only my first commit is relevant so for - and simply serves to outline the idea: use upserts instead of simple inserts to avoid key constraint errors when edges are re-asserted (which works perfectly fine with the other stores in rdflib). 

however, when I explored this idea further I did notice that this approach (which I feel is correct from a theoretical standpoint) does have one major drawback: sqlalchmey does not seem to have a way to implement upserts in a database-agnostic way which works on all supported databases. In fact, my implementation (which draws directly from sqlalchemy's reference documentation) will not work with the mysql/mariadb and sqlite backends (the latter of which is the reason you tests are failing). this doesn't matter for me (as I am using postgresql) but might be a roadblock for now. 

This is why I have started to rename my fork to "rdflib_sqlalchemy2" so noone will use my version by accident (or use projects that require the upsert implementation use your original implementation) for now.